cmake/partition_manager.cmake

```diff
@@ -204,7 +206,8 @@ if (DEFINED ext_flash_dev)
     SIZE ${num_bytes}
     BASE ${CONFIG_PM_EXTERNAL_FLASH_BASE}
     PLACEMENT start_to_end
-    DEVICE ${dev_name}
+    DEVICE ${ext_flash_dev}
+    DRIVER_KCONFIG CONFIG_NORDIC_QSPI_NOR
```

Comment:
What if I want to use a custom QSPI flash, or SPI flash? Is it possible to somehow override the driver?

Reply:
@de-nordic

Reply:
Support added for overriding